Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticsTabContent.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { useRecoilValue, useSetRecoilState } from 'recoil';
+
+import { currentProjectIdState, exportSkillModalInfoState } from '../../../../../recoilModel';
+import { navigateTo } from '../../../../../utils/navigation';
+
+import { DiagnosticList } from './DiagnosticList';
+import { useDiagnosticsData } from './useDiagnostics';
+
+export const DiagnosticsContent = () => {
+  const skillId = useRecoilValue(currentProjectIdState);
+  const setExportSkillModalInfo = useSetRecoilState(exportSkillModalInfoState);
+
+  const diagnostics = useDiagnosticsData();
+
+  return (
+    <DiagnosticList
+      diagnosticItems={diagnostics}
+      skillId={skillId}
```

Comment:
\<DiagnosticsList> shoudln't care if its skill or root bot right?

Reply:
removed the skillId concept as we discussed